scenarios.md

```diff
@@ -0,0 +1,190 @@
+# Notary Signing - Scenarios
+
+As containers and cloud native artifacts become the common unit of deployment, users want to know the artifacts in their private registries and the artifacts deployed are the same artifacts that were initially published.
+
+The [OCI TOB][oci-tob] has adopted [OCI Artifacts][artifacts-repo], generalizing container images as one of many types of artifacts that may be stored in a registry. Other artifact types currently include:
```

Comment:
There is some work being done with signing of yaml files or other type of policy based files, which are not necessarily OCI artifacts. Would something like that be in scope? 

i.e. if i want to verify the signature of an object, will the object need to be resident in an OCI registry and/or need to be an OCI object?

Reply:
> if i want to verify the signature of an object, will the object need to be resident in an OCI registry and/or need to be an OCI object?

This is proposed in Scenario 1:

> - The signing and validation of artifacts does not require a registry.
> - The signature used for validation may be hosted in a registry, or other accessible location.

Reply:
Gotcha! Thanks I missed that.

Is there a restriction of signing requiring an OCI object or are generic blobs fine? Or do you see the signing of a blob or yaml just associated with creating an "OCI Blob" type and encompassing everything else under that?

Reply:
Thanks @lumjjb, this is a good example for where we need to be crisper on artifacts, registries and signing. These are each separate things that we want to enable uniting, but not coupling. 

With Notary v2, you should be able to:
- Sign a thing, (yaml, singularity image, helm chart, YetUnknownThing) 
  - Notary v2 will provide reference examples for how you sign things, for which the tooling around the thing can implement
- [ORAS](http://github.com/deislabs/oras) will most likely adopt the generic Notary v2 signing of things. So, you could put your `.yaml` file into ORAS, sign it, and optionally push it to a registry. 
- Push it to an OCI Compliant registry, which would know how to store Notary v2 signatures. 

The OCI Blob is simply a means to storing "things". What those things are may be determined with the [OCI Artifacts](https://github.com/opencontainers/artifacts) use of `manifest.config.mediaType`

As we get further into the Notary v2 specs, and we update OCI Artifacts, we'll get more clarity for how these will work together.